src/main/kotlin/com/cognifide/gradle/aem/instance/action/DownAction.kt

```diff
@@ -0,0 +1,68 @@
+package com.cognifide.gradle.aem.instance.action
+
+import com.cognifide.gradle.aem.instance.Instance
+import com.cognifide.gradle.aem.instance.InstanceSync
+import com.cognifide.gradle.aem.instance.LocalHandle
+import com.cognifide.gradle.aem.instance.names
+import com.cognifide.gradle.aem.internal.Behaviors
+import com.cognifide.gradle.aem.internal.InstanceStateLogger
+import org.gradle.api.Project
+import java.util.stream.Collectors
+
+class DownAction(project: Project, val instances: List<Instance>) : AbstractAction(project) {
+
+    var stableTimes = config.awaitStableTimes
+
+    var stableCheck = config.awaitStableCheck
+
+    var availableCheck = config.awaitAvailableCheck
+
+    val handles = instances.map { LocalHandle(project, it) }
+
+    override fun perform() {
+        if (instances.isEmpty()) {
+            logger.info("No instances to shutdown.")
+            return
+        }
+
+        shutDownInstances()
+    }
+
+    private fun shutDownInstances() {
+        val stateLogger = InstanceStateLogger(project, "Awaiting instance(s) termination: ${instances.names}", stableTimes)
```

Comment:
stableTimes looks strange here, it should be awaitShutdownTimes / timeouting mechanism